modules/distr-tracing-rn-new-features.adoc

```diff
@@ -17,25 +17,53 @@ This release adds improvements related to the following components and concepts.
 
 This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
 
+=== {JaegerName}
 
+==== Bug fixes
 
+* link:https://issues.redhat.com/browse/TRACING-3322[TRACING-3322]: Expose 16685 port on the Jaeger Query.
+* link:https://issues.redhat.com/browse/TRACING-3312[TRACING-3312]: When deploying Service Mesh on SNO in a disconnected environment , the Jaeger Pod frequently goes into Pending state.
+* link:https://issues.redhat.com/browse/TRACING-2968[29638]: Wrong port is exposed for jaeger-production-query resulting in connection refused.
+* link:https://issues.redhat.com/browse/TRACING-3173[3173]: Jaeger operator pod restarting with OOMKilled with the default memory value.
 
+=== {TempoName}
 
+==== Enhancements
 
+* Support the operator level 4. This feature enables upgrade, monitoring and alerting of `+TempoStack+` instances and operator.
+* Add Ingress and OpenShift Route configuration for the Gateway.
+* Support managed and unmanaged state in the `+TempoStack+` custom resource.
+* Expose additional protocols (Jaeger Thrift binary, Jaeger Thrift compact, Jaeger gRPC and Zipkin) in the distributor service. When gateway is enabled only OTLP gRPC is enabled.
```

Comment:
Should we mention that the GRPC Query was also exposed?

Reply:
if it was I will include it

Reply:
I have added it as a separate item https://github.com/grafana/tempo-operator/pull/517